hrjobcontract/CRM/Hrjobcontract/Import/Parser/Api.php

```diff
@@ -597,15 +589,15 @@ private function validateField($key, $value)  {
       case 'HRJobPay-pay_cycle':
       case 'HRJobDetails-location':
       case 'HRJobDetails-end_reason':
-        $optionID = $this->getOptionID($key, $value, $optionReturnType);
+      case 'HRJobHealth-plan_type':
+      case 'HRJobHealth-plan_type_life_insurance':
+        $optionID = $this->convertOptionValue($key, $value);
```

Comment:
I feel disgusting each time I look  at the job contract importer code which I significantly updated  on my first days at compucorp , and keep saying what the hell I was doing :smile:  .. thanks for making it a bit better :D